fl4health/client_managers/fixed_sampling_client_manager.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+from flwr.server.client_manager import SimpleClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.criterion import Criterion
+
+
+class FixedSamplingClientManager(SimpleClientManager):
+    """Keeps sampling fixed until it's reset"""
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.current_sample: Optional[List[ClientProxy]] = None
+
+    def reset_sample(self) -> None:
```

Comment:
It doesn't look like this is called anywhere in the code? I think maybe you're using this to maintain the same clients sampled between evaluate and fit rounds. However, since this isn't going to be called by the server anywhere, I think the effect is that we sample at the beginning of FL and never resample again?